module/VuFindConsole/src/VuFindConsole/Command/Util/UpdateResourceMetadataCommand.php

```diff
@@ -0,0 +1,180 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see
+ * <https://www.gnu.org/licenses/>.
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFindConsole\Command\Util;
+
+use DateTime;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatter;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use VuFind\Db\Service\ResourceServiceInterface;
+use VuFind\Record\Cache;
+use VuFind\Record\Loader;
+use VuFind\Record\ResourcePopulator;
+use VuFind\RecordDriver\Missing as MissingRecord;
+
+/**
+ * Command for updating metadata in the resource table.
+ *
+ * @category VuFind
+ * @package  Console
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+#[AsCommand(
+    name: 'util/update_resource_metadata',
+    description: 'Resource metadata updater'
+)]
+class UpdateResourceMetadataCommand extends Command
+{
+    /**
+     * Constructor
+     *
+     * @param ResourceServiceInterface $resourceService   Resource service
+     * @param Loader                   $recordLoader      Record loader
+     * @param ResourcePopulator        $resourcePopulator Resource pop
+     */
+    public function __construct(
+        protected ResourceServiceInterface $resourceService,
+        protected Loader $recordLoader,
+        protected ResourcePopulator $resourcePopulator
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * Configure the command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setHelp('Updates the metadata fields of the resource table.')
+            ->addOption(
+                'batch',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Number of records to process in a single batch',
+                1000
+            )
+            ->addOption(
+                'min-age',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Minimum age of a record (in days) before it is refreshed even if metadata is not missing.'
+                . ' By default records with missing metadata are updated, but using this option allows to update all'
```

Comment:
Minor grammatical suggestion:

```suggestion
                . ' By default records with missing metadata are updated, but using this option allows updates to all'
```